airflow/providers/google/firebase/operators/firestore.py

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from collections import Iterable
+from collections.abc import Iterable
```

Comment:
It seems to me that here it will be better to use typing.Iterable. We only use it in types, not to create our own class.

Reply:
True.

Reply:
I removed it totally as there seems to be no value in this information.